assets/src/edit-story/app/file/index.js

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
@carlos-kelly @BrittanyIRL Now that the editor uses this file provider to load the fonts.json, should it also be used from the dashboard?

Reply:
It's probably best to look into that in a follow-up PR. We could also think about just re-using the same font provider.